sparse/numba_backend/_coo/core.py

```diff
@@ -851,6 +851,38 @@ def T(self):
 
     @property
     def mT(self):
+        """
+        Returns the matrix transpose as a new array.
+        The matrix transpose is the transpose of the last two dimensions, even if the array is of higher dimension
+
+        Returns
+        -------
+        COO
+            The new array with the matrix transpose.
+
+        See Also
+        --------
+        - [`sparse.COO.transpose`][] :
+            A method where you can specify the order of the axes.
+        - [`numpy.ndarray.mT`][] :
+            Numpy equivalent property.
+
+        Examples
+        --------
+
```

Comment:
```suggestion
```

Unless the newline is needed for proper rendering, I'd remove it.

Reply:
Done.